### ISSUE TITLE
"User id not found" after verifying the Key with gmail in Firefox

### ISSUE THREAD

**Issue by skriesch** [mailvelope/keyserver#75]
I am using gmail and wanted to create a new key pair for sending emails.
I can receive the email to verify the key. After entering the password I receive a "User id not found" in the Firefox.
After that I tested the same procedure with Chromium. It works there.

Link to "User id not found": https://keys.mailvelope.com/api/v1/key?op=verify&keyId=e060293bba13aefb&nonce=5c42fe37d73d48b2ae6f82f9bc457515

Firefox version: 60.2.2esr (64-Bit)
Operating System: openSUSE Leap 15.0

**Comment by skriesch** [comment]
This issue exists on other Linux distributions like Ubuntu, too.

**Comment by toberndo** [comment]
The verification link works only once, so after you have verified a certain email address, a subsequent click on the verification link will lead to the "User id not found" message. Could it be that this happened in your case?
Probably we should distinguish the "not found" case from the "already verified" case and provide a more meaningful message.

**Comment by skriesch** [comment]
No. I wanted to verify my email address and wanted to accept the verification link once. It forwarded me to "User id not found". You can test it with Linux distributions like Ubuntu and openSUSE. Other people had the same issue in our group.
After that I tried it with Chromium, created a new key pair and the verification has worked there (same procedure).

**Comment by daudo** [comment]
confirming this behaviour as well. Opening the confirmation link in FF 66.0.3 in Manjaro Linux doesn't work ("User id not found"), whereas it works flawlessly when I try the same in chrome/chromium.

**Comment by Volker-K** [comment]
Same behaviour here, the key can't have been confirmed before, otherwise it would be found on the key server.

**Comment by stradarius** [comment]
With Firefox 89.0.2 64bit on Linux Mint I cannot reproduce this error - neither with the demo-server nor with the current code in a local deployment. Works as expected.

**Comment by rappertomate** [comment]
> confirming this behaviour as well. Opening the confirmation link in FF 66.0.3 in Manjaro Linux doesn't work ("User id not found"), whereas it works flawlessly when I try the same in chrome/chromium.

Basically this in my case as well. FF 89.0 (64-bit) on Manjaro gives me this error, Chromium doesn't.

**Comment by ivanisconfused** [comment]
still experiencing this - FF 125.0.2 on Ubuntu 24.04
